packages/design-system/src/components/contextMenu/menuItem.js

```diff
@@ -95,26 +86,35 @@ export const MenuItem = ({
         </Tooltip>
       );
     }
+
+    /* Shortcut title to be read by screen reader. */
+    const visuallyHiddenContent = shortcut?.title ? (
+      <VisuallyHidden>{shortcut?.title}</VisuallyHidden>
```

Comment:
Just a nit, isn't this null check redundant since we have already checked that `shortcut.title`  exists on line 91?